plugins/modules/sda_fabric_virtual_networks_workflow_manager.py

```diff
@@ -108,7 +108,7 @@
             type: list
             elements: dict
             suboptions:
-              site_name:
+              site_name_hierarchy:
```

Comment:
Can't we say site_name itself like in the other module?

Reply:
will decide on this.